### ISSUE TITLE
Pack Gear lever as AppImage

### ISSUE THREAD

**Issue by kem-a** [mijorus/gearlever#186]
Just a small request: it would be fantastic if the app that manages AppImages could actually be available as an AppImage itself.

Requiring Flatpak to install an app meant to handle AppImages feels counterintuitive and defeats the purpose of using AppImages in the first place.

**Comment by tbone2k-git** [comment]
Yes, I agree! o) I guess we are not the first people coming here with this suggestion? o)

I came here as an AppImage user on a handful of very small Linux setups (storage space below 2GB). AppImages allow me to run and maintain these little setups with ease. Would be cool if GearLever came as an AppImage as well! o)

Thank you! o)

**Comment by tbone2k-git** [comment]
Since this seems to be a Python project, is there a way to run the Python code directly, without any AppImage or Flatpak packaging involved? I'm not sure about how Python dependencies are handled, but in theory.. running the Python code would work for me as well, keeping it simple!?

**Comment by RokeJulianLockhart** [comment]
> https://github.com/mijorus/gearlever/issues/186#issue-2643721735

@kem-a, surely this is a duplicate of https://github.com/mijorus/gearlever/issues/83#issue-2317540046?

**Comment by hotlittlewhitedog** [comment]
@kem-a There is the unofficial AppImage version compiled by @Ivan-hc on the [homepage](https://github.com/mijorus/gearlever?tab=readme-ov-file)
